pkg/apis/application/v1alpha1/types.go

```diff
@@ -464,6 +466,9 @@ type AppProjectSpec struct {
 	Description string `json:"description,omitempty" protobuf:"bytes,3,opt,name=description"`
 
 	Roles []ProjectRole `json:"roles,omitempty" protobuf:"bytes,4,rep,name=roles"`
+
+	// ClusterResources contains list of cluster level group kinds which are accessible by project apps
+	ClusterResources []metav1.GroupKind `json:"clusterGroupKinds,omitempty" protobuf:"bytes,5,opt,name=clusterGroupKinds"`
```

Comment:
Thinking further, how do you think we should handle whitelisting/blacklisting of namespaced resources? For example, especially with the advent of CRDs I believe that eventually, cluster operators will need to restrict what can be permitted even with expensive namespaced resources (e.g. ServiceBroker, DaemonSets, etc...)

Reply:
As we discussed offline renamed ClusterResources to `ClusterResourceWhiteList` and introduced `NamespaceResourceBlackList`.